### ISSUE TITLE
2.0.43 reverted some of the waiters like networkinterfaceavailable

### ISSUE THREAD

**Issue by eniskonuk** [aws/aws-sdk-ruby#818]
Is there a reason why the waiters that I had put into the code via a pull request got reverted back? Our code just stopped working when we updated to 2.0.43. Can these be merged back in?
Thx
Enis

**Comment by trevorrowe** [comment]
I'll take a look at getting this fixed. My guess is merged changes did not get merged upstream. When @awood45 pulled the updated API files for Thursday's release, is probably when these got reverted. My apologies.

**Comment by trevorrowe** [comment]
It looks like it was merged and shipped with v2.0.39 and then lost in v2.0.40. The offending commit is here: https://github.com/aws/aws-sdk-ruby/commit/f0fad0a038a4f81b0a697f7c367fd2735d9dd698. I've gone back and merged this upstream. The fix will be shipped today. I also audited all of the other changes made in your pull-request. The other changes have been merged up-stream and are intact.

Sorry for the confusion and inconvenience. We are currently looking at the process on how we update and maintain some of these shared metadata files. As the Ruby SDK is not the source, we need a better process to ensure we don't have regressions like this in the future.
